frontend/src/metabase/admin/permissions/components/PermissionsEditor/PermissionsEditorLegacyWarning.tsx

```diff
@@ -0,0 +1,40 @@
+import { Link } from "react-router";
+import { t } from "ttag";
+
+import { useToggle } from "metabase/hooks/use-toggle";
+import { colors } from "metabase/lib/colors";
+import { Icon, Alert, Anchor, Text, Box } from "metabase/ui";
+
+export const PermissionsEditorLegacyNoSelfServiceWarning = () => {
+  const [isExpanded, { toggle }] = useToggle(false);
+
+  return (
+    <Box mt="md" mb="sm" mr="2.5rem">
+      <Alert icon={<Icon name="warning" size={16} />} color="accent5">
+        <Text fw="bold">
+          {t`The No self-service access level for View data is going away.`}
+          {!isExpanded && (
+            <>
+              {" "}
+              <button onClick={toggle}>
+                <Text fw="bold" color={colors.accent7}>{t`Read more`}</Text>
+              </button>
+            </>
+          )}
+        </Text>
+
+        {isExpanded && (
+          <Text>
+            {t`In a future release, if a group’s View data access for a database (or any of its schemas or tables) is still set to No self-service (deprecated), Metabase will automatically change that group’s View data access for the entire database to Blocked. We’ll be defaulting to Blocked, the least permissive View data access, to prevent any unattended access to data.`}{" "}
```

Comment:
Should "No self-service" be in smart quotes for consistency?

I'm unsure if "Metabase will automatically..." and "We'll be defaulting..." are consistent with how we normally talk about the app within the app

Reply:
I copy/pasted this from somewhere... I best look into that too.

Changed.

I'll ask product for more feedback on this copy. This copy is being sent out in a few places so maybe we need to refine that messaging all around.